src/utils/tx-list.ts

```diff
@@ -49,18 +49,20 @@ export function _getRecoveryCancellations(moduleAddress: string, transactions: A
 export function groupRecoveryTransactions(queue: Array<TransactionListItem>, recoveryQueue: Array<RecoveryQueueItem>) {
   const transactions = queue.filter(isTransactionListItem)
 
-  return recoveryQueue.reduce<Array<Array<Transaction | RecoveryQueueItem>>>((acc, item) => {
-    acc.push([item])
+  return recoveryQueue.reduce<Array<Transaction | RecoveryQueueItem | Array<Transaction | RecoveryQueueItem>>>(
```

Comment:
Nit: This type is hard to read.

We could have an alias `type QueuedItem =  Transaction | RecoveryQueueItem`
And then `Array<QueuedItem | Array<QueuedItem>>`